docs/docs/api/04-Relations/and.md

```diff
@@ -3,14 +3,16 @@ displayed_sidebar: apiSidebar
 ---
 # and
 
+<span class="theme-doc-version-badge badge badge--secondary"></span>
+
 Logic and operator
 
 **Examples:**
 ```text 
 example scene: 
  ---------------   ----------------
  |  icon user  |   |  icon search |
- ---------------   ----------------```
+ ---------------   ---------------n```
```

Comment:
@JohannesDienst-askui  ?

Reply:
This is an error in the askui-inference MethodDocu.body. Will fix it there and regenerate! Good catch!